styleguide/source/_patterns/02-organisms/00-global/04-search-results-columns/search-results-columns.twig

```diff
@@ -0,0 +1,17 @@
+<div class="layout layout_two_up-large search-results-columns">
+  <div class="layout_two_up-large-primary">
```

Comment:
This can't use layout modules as they exist now. This organism and the needs to list all of the search results as siblings and then use .search-results-columns to control the layout. This class will be swapped for .search-results-list to switch the layout.